src/index.ts

```diff
@@ -28,30 +28,44 @@
 //     unmarshalled from an incoming request.
 
 import * as minimist from 'minimist';
+
 import {
   ErrorHandler,
   FunctionSignatureType,
   getServer,
   getUserFunction,
 } from './invoker';
 
+// Supported command-line flags
+const FLAG = {
+  PORT: 'port',
+  TARGET: 'target',
+  SIGNATURE_TYPE: 'signature-type', // dash
+};
+
+// Supported environment variables
+const ENV_VARIABLE = {
+  PORT: 'FUNCTION_PORT',
+  TARGET: 'FUNCTION_TARGET',
+  SIGNATURE_TYPE: 'FUNCTION_SIGNATURE_TYPE', // underscore
```

Comment:
While we are refactoring this, please replace `FUNCTION_SIGNATURE_TYPE` in `FUNCTION_SIGNATURE_TYPE must be one of 'http' or 'event'.` message below with `Function signature type`, as it can refer to either flag or env var.